src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
That is reason why C++ allows override function.
Good idea, to use call overriden inside template, will implement